zerver/tests/test_embedded_bot_system.py

```diff
@@ -34,8 +34,9 @@ def test_pm_to_embedded_bot(self) -> None:
         self.assertEqual(last_message.sender_id, self.bot_profile.id)
         display_recipient = get_display_recipient(last_message.recipient)
         assert isinstance(display_recipient, list)
-        self.assert_length(display_recipient, 1)
+        self.assert_length(display_recipient, 2)
         self.assertEqual(display_recipient[0]["email"], self.user_profile.email)
+        self.assertEqual(display_recipient[1]["email"], self.bot_profile.email)
```

Comment:
This change looks to be a change visible to API clients -- what code change causes it? 

In general our goal for this migration should be to cut over the database separately from cutting over the API, and ideally to avoid API changes at all, since those require coordination with clients. This particular API change would be easy to manage, since it's just with the embedded bots system not in production use, but I am not certain what other code paths might be affected by the root cause of this change.